README.md

```diff
@@ -23,6 +23,14 @@ For more details on Microsoft Azure Storage, please visit the <a href="https://a
 1. Activate the plugin through the 'Plugins' screen in WordPress.
 1. Use the Settings->Microsoft Azure screen to configure the plugin.
 
+## Setup
+
+In order for uploads to properly make it to Azure Storage, the following needs to be added to the wp-config.php file:
+```
+define( 'FS_METHOD', 'direct' );
+define( 'DISALLOW_FILE_MODS', 'true' );
```

Comment:
Based on https://wordpress.org/support/article/editing-wp-config-php/#disable-plugin-and-theme-update-and-installation there should be Boolean value. I don't know how Wordpress uses string value of true.